src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -277,7 +301,7 @@ private void CreateRequestBuilderClass(CodeNamespace currentNamespace, OpenApiUr
                 CreateRequestBuilderClass(targetNamespace, childNode, rootNode);
             });
         }
-
+        private static readonly string coreInterfaceType = "IHttpCore";
```

Comment:
nit: consider hoisting this above L211 since this string is used in the method starting at L211.

Reply:
done